docs/src/faqs/migrate_from_fnapi.rst

```diff
@@ -1,229 +1,76 @@
-How Do I Migrate from FnAPI to ``setup.py``?
+How Do I Migrate from ``setup.py`` to FnAPI?
 ============================================
 
-The FnAPI (pronounced "fun API") is what allows Klio to run Docker images on Dataflow. However,
-it's still considered experimental. The fully-supported way to run a job that has dependencies
+The FnAPI (pronounced "fun API") is what allows Klio to use custom Docker images
+on Dataflow workers.
+However, it's still considered experimental.
+The fully-supported way to run a job that has dependencies
 (both Python and OS-level dependencies) is via `setup.py <https://beam.apache.org/documentation/
 sdks/python-pipeline-dependencies>`_.
 
-Below describes what changes need to be made to an existing job to move from the FnAPI to
-``setup.py``.
+Below describes what changes need to be made to an existing job to move from
+``setup.py`` to FnAPI.
 
-Creating a new Klio job that does not use the FnAPI from the start via:
+Creating a new Klio job that does not use ``setup.py`` from the start via:
 
 .. code-block:: console
 
-    $ klio job create --use-fnapi=false
-
-Limitations and Warnings
-------------------------
-
-* Currently, Klio in non-FnAPI mode does not yet support jobs with multiple configuration files. Support is planned.
-* ``pipeline_options.requirements_file`` configuration for `pipeline dependencies`_ **will not work** for Klio jobs. While klio will honor that configuration value for Dataflow to pick up, declaring requirements in ``setup.py`` is needed because a Klio job inherently has multiple Python files.
-* While Klio will still upload the worker image to `Google Container Registry`_ when running/deploying a job, Dataflow will *not* use the image. It is good practice to upload the worker image to ensure repeatable builds, but in the future, an option will be added to skip the upload.
-
```

Comment:
I'm cool with removing this section, but I still think it's valuable information. And I'm not sure where to put it. They can probably be split up and put into different, relevant places, maybe? Any thoughts?

Reply:
Hmm, maybe a section under `Anatomy` called `Dependency Management`? This section would detail and explain the differences between the two options. I could also see this information going into the `FAQ` if it seems inappropriate for the `User Guide`.

Reply:
Ugh, also on re-reading this I just realized that I screwed up a bit (not in a breaking way, but just in an accurate messaging way) with [this PR that I just merged in](https://github.com/spotify/klio/pull/127/files) since it makes it seem like you can simply provide a `requirements.txt` file.

Reply:
Update - took a stab at creating a separate FAQ for this, called "What is needed to use setup.py to package dependencies in Klio"?